### DISCUSSION TITLE
Can't connect wallet on mobile: Navigation is blocked

### DISCUSSION THREAD

**Discussion by joshi-kaushal** [rainbow-me/rainbowkit#1956] [Q&A]
I am using RainbowKit V2 in my app. It's working fine on the desktop. However, I cannot connect my wallet to mobile browser.

I used [remote debug android device ](https://developer.chrome.com/docs/devtools/remote-debugging) and could see console logs and network requests. One of the logs says that the navigation is blocked. Here's the warning:

> Navigation is blocked: wc:{alphaneumeric-string}@2/wc?requestId={request-id}&sessionTopic={session-topic}

What am I missing? I am providing versions of related dependencies and apps. I cannot create a demo app because it includes some custom authentication as well. Please let me know if you need any extra info.

- @rainbow-me/rainbowkit 2.0.5,
- viem: 2.9.9,
- wagmi: 2.5.19
- Android version: Android 14
- Chrome version: 124.0.6367.82 (tried on different browsers including Safari)

**Comment by 0xmkh** [comment]
@joshi-kaushal Which wallet are you using ? Can you also share a minimal reproducible example ?
